guides/doc-Administering_Red_Hat_Satellite/topics/snip_katello-only.adoc

```diff
@@ -0,0 +1,2 @@
+[NOTE]
+This section is only for Katello plugin users.
```

Comment:
I'd swear we already have something like that. Interesting.

Reply:
We had for the provisioning guide but then that was changed by work done to use one set of prereqs for all Compute Resources!